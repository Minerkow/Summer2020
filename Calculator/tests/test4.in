pt  = 824 -((226 /496 )-974 *76 )-(533 /504 -293 *112 *282 /431 );
po = ?;
lgp  = po *((951 -((162 *((653 )/(279 +(872 -(po ))-94 )*509 )))/168 )/210 /540 );
s  = po +((651 -59 )*((((((709 /480 +(((591 /504 )*(910 ))-868 )))))-705 )-276 ))*(602 +514 *57 )/(795 *860 +26 /924 -265 +967 );
pt  = 12 *399 /pt +(141 /(667 )*362 +527 );
y  = 329 +132 -pt /247 *((20 /693 ));
print pt;
print lgp;
print s;
print y;