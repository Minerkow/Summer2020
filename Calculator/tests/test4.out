527
0
2055
461
